AndroidTriviaNavigation/app/src/main/java/com/example/android/navigation/TitleFragment.kt

```diff
@@ -50,13 +50,13 @@ class TitleFragment : Fragment() {
     }
 
 
-    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
         super.onCreateOptionsMenu(menu, inflater)
-        inflater?.inflate(R.menu.options_menu,menu)
+        inflater.inflate(R.menu.options_menu,menu)
     }
 
-    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
-        return NavigationUI.onNavDestinationSelected(item!!,view!!.findNavController())
```

Comment:
code format this file? I think some extra spaces in there could help with readability

Reply:
this is deleted code